test/type/unit/Unit.test.js

```diff
@@ -1040,4 +1043,92 @@ describe('Unit', function() {
       assert.equal(new Unit(1, 'eV')    .equals(new Unit(1.602176565e-19, 'J')), true);
     });
   });
+
+  describe('createUnitSingle', function() {
+    it('should create a custom unit from a string definition', function() {
+      Unit.createUnitSingle('widget', '5 kg bytes');
+      assert.equal(new Unit(1, 'widget').equals(new Unit(5, 'kg bytes')), true);
+      Unit.createUnitSingle('woggle', '4 widget^2');
```

Comment:
Maybe it makes sense to move `Unit.createUnitSingle('woggle', '4 widget^2');` into a `before(...)` block since it's used in multiple tests?